cms/urls.py

```diff
@@ -287,7 +287,8 @@
     ]
 
 # API docs.
-urlpatterns += make_docs_urls(api_info)
+# Appsembler: Disabled for security reasons. Should be back after upgrading drf-yasg. See RED-2861 for details
+# urlpatterns += make_docs_urls(api_info)
```

Comment:
a better pattern would be the following:
```suggestion
if settings.FEATURES.get('TAHOE_ENABLE_API_DOCS_URLS', False):
    urlpatterns += make_docs_urls(api_info)
```

It adds a sense of "feature" to the change we're doing and make it easier to search by the feature flag to identify different usage and affect of the feature.

Reply:
done